lib/code_corps/cloudex/cloudinary_url.ex

```diff
@@ -7,5 +7,15 @@ defmodule CodeCorps.Cloudex.CloudinaryUrl do
   end
   def for(public_id, options, _version, _default_color, _type) do
     @cloudex.Url.for(public_id, options)
+    |> add_html_scheme
   end
+
+  defp add_html_scheme(generated_url) do
```

Comment:
So sorry for all the back and forth here, but I do think that `html` is not quite right here. I think it should be `add_uri_scheme`. Thoughts?

Reply:
its all good I meant to change that and forgot